kolibri/plugins/coach/assets/src/views/ClassLearnersListPage.vue

```diff
@@ -52,7 +52,7 @@
         <template #tbody>
           <tbody>
             <tr
-              v-for="learner in learnerMap"
+              v-for="learner in learnersWithLearnOnlyDevices"
```

Comment:
So, we don't actually need to change this condition here - that's out of scope for this issue. There is handling within this component already for managing learners who haven't synced yet, which we want to preserve